app/views/occupation_standards/show.html.erb

```diff
@@ -160,87 +160,66 @@
                 <label class="label py-3 mr-4 md:mr-2 order-first text-center hover:border-b-3px hover:border-gray-400 active:bg-gray-100" for="tab2">Related Instruction Outline</label>
+
                 <div class="-mt-0.5 border-t-2 border-seafoam-700 border-opacity-30 panel hidden w-full" tabindex="0"> <!-- begin content tab 2 -->
-                      <div class="flex justify-between p-4">
+                    <div class="flex justify-between p-4">
                         <div>
                             <h2 class="py-2 text-2xl font-bold">Related Instruction Content</h2>
                             <p>Training Provider(s):</p>
                             <div class="my-3 flex flex-col md:flex-row">
-                                <div class="mr-6 mb-4 flex md:items-center">
-                                    <div>
-                                        <img class="mr-2 h-7 w-auto rounded" src="https://flowbite.com/docs/images/examples/image-1@2x.jpg" alt="organization logo">
-                                    </div>
-                                    <div class="font-bold">Laney College</div>
-                                </div>
-                                <div class="mr-6 mb-4 flex md:items-center">
-                                    <div>
-                                        <img class="mr-2 h-7 w-auto rounded" src="https://flowbite.com/docs/images/examples/image-1@2x.jpg" alt="organization logo">
+
+                                <% Organization.related_instructions_organizations(@occupation_standard).pluck(:title).each do |related_instruction_organization| %>
+                                    <div class="mr-6 mb-4 flex md:items-center">
+                                        <div>
+                                            <img class="mr-2 h-7 w-auto rounded" src="https://flowbite.com/docs/images/examples/image-1@2x.jpg" alt="organization logo">
+                                        </div>
+                                        <div class="font-bold"><%= related_instruction_organization %></div>
                                     </div>
-                                    <div class="font-bold">ABC Technical School</div>
-                                </div>
+                                <% end %>
                             </div>
                         </div>
 
                         <div class="flex items-center">
                             <div class="flex flex-col md:flex-row">
-                                <div class="m-1.5 w-28 rounded bg-pear-100 px-1 py-3 text-center">
-                                    <div class="text-3xl font-bold text-pear-700">600</div>
-                                    <div class="text-sm font-bold text-pear-800">RSI hours</div>
-                                </div>
+                                <% unless @occupation_standard.related_instructions_hours.zero? %>
```

Comment:
Maybe: `if @occupation_standard.related_instructions_hours.positive?` But I'm fine either way. Zero is quite clear.